src/collections/ipcCollection.js

```diff
@@ -196,6 +196,12 @@ function facilityBridge() {
         });
     });
 
+    ipcMain.on("Facility:getFacilitiesByState", (e, arg) => {
```

Comment:
What are `e` and `arg`?

Reply:
event and arguments. I think thats the convention for those names in js.

Reply:
This conversation is continued at https://github.com/7GCraft/ces-manager/pull/98#discussion_r920807481